### PR TITLE
added pipeline to run customer-admin-permissions

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Motivation
https://issues.jboss.org/browse/INTLY-1266

## Why
To allow to run the customer-admin-permissions test on Jenkins

## Verification Steps
1. To make sure that the pipeline definition is syntactically correct:

```jenkins-jobs --conf jenkins_jobs.ini update jobs/integr8ly/customer-admin-permissions-test.yaml```

2. Go to https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/customer-admin-permissions-test/1/console and confirm that the pipeline has successfully completed

## Checklist:

- [x] Code has been tested locally by PR requester

## Progress

- [x] Finished task

### CODE REVIEW

**Comment by tremes** [comment]
@pawelpaszki Please add your new pipelines to https://github.com/integr8ly/ci-cd/blob/master/scripts/configure_jenkins.sh so that can create or recreate it automatically.

File: jobs/integr8ly/customer-admin-permissions-test.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            node('cirhos_rhel7'){
+
+                Boolean publishTestResults = true
+
+                stage('Clone QE repository') {
+                    dir('.') {
+                        git branch: "${BRANCH}", url: "${REPOSITORY}"
+                    }
+                }
+                stage('Run the alert checking tests'){
```
Comment (tremes) [inline jobs/integr8ly/customer-admin-permissions-test.yaml]: This is not correct name right?:-)

Comment (pawelpaszki) [reply jobs/integr8ly/customer-admin-permissions-test.yaml]: good catch. fixed

**Comment by pawelpaszki** [comment]
> @pawelpaszki Please add your new pipelines to https://github.com/integr8ly/ci-cd/blob/master/scripts/configure_jenkins.sh so that can create or recreate it automatically.

Done

**Comment by pawelpaszki** [comment]
@tremes @trepel can you have a final look?

**Comment by tremes** [comment]
LGTM

**Pull request merged**